llvm/include/llvm/ADT/APFloat.h

```diff
@@ -146,6 +146,12 @@ struct APFloatBase {
   /// A signed type to represent a floating point numbers unbiased exponent.
   typedef int32_t ExponentType;
 
+  enum RadixAndFormat {
+    BaseTwo,
+    BaseTenBID,
+    BaseTenDPD
+  };
```

Comment:
What's the rationale for combining radix and format into one enum?

Reply:
It was a trial format but if we are not going to have separate types for BID/DPD then we don't need that anymore.